.eslintrc.js

```diff
@@ -18,4 +18,8 @@ module.exports = {
 			},
 		},
 	},
+
+	rules: {
+		'vue/no-reserved-component-names': 'off', // @nextcloud/vue has Button, Content components restricted by rule
```

Comment:
@JuliaKirschenheuter There is no need to disable this rule, if you import the components differently.
Just import the respective components with their correct name like this:
```js
import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
```